tests/integerationTests/PostArticle.test.jsx

```diff
@@ -0,0 +1,75 @@
+import "babel-polyfill";
+import React from "react";
+import { mount, shallow } from "enzyme";
+import configureStore from "redux-mock-store";
+import thunk from "redux-thunk";
+
+import PostArticle from "../../src/components/article/PostArticle";
+describe("<PostArticle />", () => {
+  document
+    .getElementsByTagName("head")[0]
+    .appendChild(document.createElement("script"));
+  const middlewares = [thunk];
+  const mockStore = configureStore(middlewares);
+  const initialUserState = {
+    article: {
+      title: "",
+      imageUrl: "",
+      description: "",
+      body: "",
+      audioUrl: "",
+      tags: []
+    }
+  };
+  const preventDefault = jest.fn();
+  const store = mockStore({ ...initialUserState });
+  const provider = shallow(<PostArticle store={store} />);
+  const PostArticleComponent = mount(<PostArticle store={store} />);
+  it("renders <Provider/> correctly", () => {
+    expect(provider).toMatchSnapshot();
+  });
+  const formWrapper = provider.dive().find("main");
+  //   const widgetFn = provider.dive().instance().handleOnClick();
+
+  it("should render main", () => {
+    expect(formWrapper.find("main.mt-5").length).toEqual(1);
+  });
+
+  test("test function ", async () => {
+    // const result = [{ }];
```

Comment:
could you remove this commented out code

Reply:
@shakirandagire I have worked on this